dashboard/test/lib/python_test.rb

```diff
@@ -0,0 +1,84 @@
+require 'test_helper'
+require 'python/python'
+
```

Comment:
This file is mostly a rename of `pycall_test.rb` with the addition of putting python calls inside a `Python.run()` block, which made git think it was a new file not a rename.